packages/database/lib/schema.ts

```diff
@@ -43,20 +56,29 @@ export const daos = pgTable('daos', {
   daoAcct: pubkey('dao_acct').primaryKey(),
   name: varchar('name').notNull(),
   url: varchar('url').notNull(),
+  xAccount: varchar('x_account'),
```

Comment:
Not sure if these should be notNull or what, I want to be flexible, but also.......

Reply:
I think if it's not enforced on chain, then it's fine to leave blank as we still want to index this stuff...